src/routes/ar-io.ts

```diff
@@ -219,3 +221,35 @@ arIoRouter.post(
     }
   },
 );
+
+arIoRouter.post(
+  '/ar-io/admin/export-parquet',
+  express.json(),
+  async (req, res) => {
+    try {
+      const { outputDir, startHeight, endHeight, maxFileRows } = req.body;
+
+      if (!outputDir || !startHeight || !endHeight || !maxFileRows) {
+        res.status(400).send('Missing required parameters');
+        return;
+      }
+
+      const parquetExporter = new ParquetExporter({
+        log,
+        duckDbPath: './data/duckdb/tags.duckdb',
+        sqliteDbPath: './data/sqlite/bundles.db',
```

Comment:
nit: put these behind env vars

Reply:
I didn't add a env var specifically for it as we handle paths a little bit different, but I added `DUCKDB_DATA_PATH` on docker-compose